.github/workflows/nightly.yaml

```diff
@@ -37,4 +37,35 @@ jobs:
           files: |
             microshift-linux-amd64
             microshift-linux-arm64
-            release.sha256
\ No newline at end of file
+            release.sha256
+  
+  build-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Install required packages
+        run: sudo apt install build-essential qemu-user qemu-user-static  
+
+      - name: launch amd build
+        run: make build-containerized-all-in-one-amd64-nightly
+
+      - name: launch arm build
+        run: make build-containerized-all-in-one-arm64-nightly
+
+      - name: launch arm iptables build
+        run: make build-containerized-all-in-one-iptables-arm64-nightly
+
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASS }}
+
+      - name: push assets
+        run: |
+          for i in `podman images --sort repository --format "{{.Repository}}:{{.Tag}}" quay.io/microshift/microshift-aio`; do
+            podman push $i
```

Comment:
We should probably publish them under a single multiarch manifest, see:

https://github.com/redhat-et/microshift/blob/ce1103717f3b91a1d5537a7473effe04fde2d74e/packaging/images/components/build.sh#L91
https://github.com/redhat-et/microshift/blob/ce1103717f3b91a1d5537a7473effe04fde2d74e/packaging/images/components/build.sh#L92
https://github.com/redhat-et/microshift/blob/ce1103717f3b91a1d5537a7473effe04fde2d74e/packaging/images/components/build.sh#L126
and 
https://github.com/redhat-et/microshift/blob/ce1103717f3b91a1d5537a7473effe04fde2d74e/packaging/images/components/build.sh#L55